Kernel/Modules/AdminCustomerUser.pm

```diff
@@ -371,6 +371,7 @@ sub Run {
                 );
             }
 
+            $CurrentUserData{UserPassword} //= '';
             if ( $CurrentUserData{UserPassword} ne $GetParam{UserPassword} ) {
```

Comment:
Please don't change the variable, instead use:
if ( ( undef // "123" ) eq "123" ) {print "klappt\n"}